modules/apiserversource-kn.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="apiserversource-kn_context"]
+= Using the ApiServerSource with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete an ApiServerSource using `kn` commands.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have created the `default` broker in the same namespace that the ApiServerSource will be installed in.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. Create a service account, cluster role and cluster binding for the ApiServerSource.
++
+You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default <1>
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
The ClusterRole and ClusterRoleBinding requires a cluster-admin. 

The tutorial should work with just the same with Role and RoleBinding instead, as the serviceaccount only needs access to the same namespace it lives in.

Reply:
Made some changes to reflect this - lmk if it looks OK to you :)